api/src/main/java/org/neo4j/ogm/config/Configuration.java

```diff
@@ -47,6 +47,14 @@
 public class Configuration {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Configuration.class);
+
+    /**
+     * The class loader that should be used for loading domain entities, custom converters etc. This shall be used for
+     * driver, domain objects and custom converters. Resourced that are needed before this shall be loaded via the
```

Comment:
Second sentence is just a repetition of the first one, or?

Reply:
yep